src/js/actions/transform.js

```diff
@@ -504,6 +504,11 @@ define(function (require, exports) {
             headlights.logEvent("edit", "layers", "swap-non-artboard");
         }
 
+        var transaction = descriptor.beginTransaction(options),
```

Comment:
style nit: it seems a little awkward how `options` is dangling a dozen or so lines above.